tests/data/simple_prismatic_joints.urdf

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<robot name="simple_prismatic_joints">
+  <link name="BaseLink">
+    <visual>
+      <origin rpy="0 0 0" xyz="0.0 0 0"/>
+      <geometry>
+        <box size="0.2 0.2 0.2"/>
+      </geometry>
+    </visual>
+  </link>
+  <link name="Arm:1">
```

Comment:
Are you intentionally using invalid prim names in all these files? That seems like something we'd only need to test once in a dedicated file & the rest of the tests could use Arm1 or Arm_1